config/samples/logstash/logstash.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: elasticsearch.k8s.elastic.co/v1
```

Comment:
@kaisecheng should this sample include how to use monitoring? And or, add a new sample that shows how to use monitoring?